src/gpio.rs

```diff
@@ -19,6 +19,9 @@ use crate::peripheral::{Peripheral, PeripheralRef};
 
 pub use chip::*;
 
+#[cfg(feature = "nightly")]
```

Comment:
Does not need this protection, as `InputFuture` does not use nightly features. However does need the (rather annoying) `riscv-ulp-hal` protection for now, until we finally pull of the ULP hal out of this crate.

Reply:
It also needs `alloc` protection too, because of the alloc requirement with subscribed pins